cmd/dfdaemon/options/options.go

```diff
@@ -1,12 +1,5 @@
 package options
```

Comment:
can we move this package to `cmd/dfdaemon/app/options`?

Reply:
Done.